less/index.less

```diff
@@ -54,6 +54,225 @@ table {
 }
 
 html, body {
+	
     height: 100%;
 }
+ .displayPreference() {
+	display: flex;
+	padding: 1rem;
+	margin: 1rem;
+ }
 
+ .setPaddingAndMarginTo1Rem() {
+	padding: 1rem;
+	margin: 1rem;
+ }
+
+ @setAlignmentToBase: baseline;
+ @mobileBreakpoint: 500px;
+ @mobileRootFontSize: 80%;
+ @desktopRootFontSize: 100%;
+
+:root {
+	font-size: @desktopRootFontSize;
+	width: 62.5%;
+	margin: 0 auto;
+
+	@media(max-width: @mobileBreakpoint) {
+		font-size: @mobileRootFontSize;
+		// width: 70%;
+	  }
+}
+
+.container {
+	margin: 0 auto;
```

Comment:
**Where is your nesting?**
**The goal** if to have every selector nested inside of this main selector. while each of them can have their media queries nested within them as well.